aptos-move/aptos-native-interface/src/context.rs

```diff
@@ -119,8 +119,13 @@ impl<'a, 'b, 'c, 'd> SafeNativeContext<'a, 'b, 'c, 'd> {
         self.enable_incremental_gas_charging = enable;
     }
 
-    /// Returns true if the aggregator snapshots feature is enabled.
-    pub fn aggregator_snapshots_enabled(&self) -> bool {
-        self.get_feature_flags().is_aggregator_snapshots_enabled()
+    /// Returns true if the aggregator API feature is enabled.
+    pub fn aggregator_api_enabled(&self) -> bool {
+        self.get_feature_flags().is_aggregator_api_enabled()
```

Comment:
The feature flags `aggregator_api_enabled`, `aggregator_execution_enabled` sound very similar to me. What's the difference? 
Why is `aggregator_snapshots_enabled` feature flag removed now? Can we remove feature flags?

Reply:
I've repurposed aggregator_snapshots_enabled to also gate aggregator_snapshot , and called it aggregator_api_enabled.

that's assuming we don't create 1.7.x for tokens and enable snapshots before. if we do, we will need to flags for that.

I think difference in the name of aggregator_api_enabled and aggregator_execution_enabled explains the difference (added some comments to where flags are defined) - basically first one - API - allows for aggregators_v2.move to be used (i.e. ecosystem contracts can be updated.

Second one - execution - actually makes aggregators concurrent